client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -111,8 +93,8 @@ class ConversationCaterpillarComponent extends React.Component {
 		const gravatarSmallScreenThreshold = MAX_GRAVATARS_TO_DISPLAY / 2;
 
 		return (
-			<Card className="conversation-caterpillar" onClick={ this.handleTickle }>
-				<div className="conversation-caterpillar__gravatars">
+			<Card className="conversation-caterpillar">
```

Comment:
Is there a reason why we are using `card` here instead of a `div`? I think a `div` makes more sense since we're eliminating all the styles attached to a `card`.

Reply:
i don't think theres a reason!
cc @bluefuton

Reply:
Swap to `div` makes sense 👍